server/modules/record/service/recordService.js

```diff
@@ -112,7 +66,7 @@ const checkIn = async (user, surveyId, recordUuid, draft) => {
     if (RecordThreads.isZombie(recordUuid)) {
```

Comment:
I would move this check to RecordServiceThreads; the import of RecordsThreads should be only there

Reply:
yes, good point. the methods RecordThreadsMap methods used by RecordService can be used through RecordServiceThreads

Reply:
Ok